src/scrapers/CourseScraper.js

```diff
@@ -0,0 +1,90 @@
+import * as cheerio from'cheerio';
+
+
+//getCourse('https://tvarkarasciai.vu.lt/mif/ajax_fullcalendar_events/informatika-1k-1gr-2024/group/265/?start=2025-01-06T00%3A00%3A00%2B02%3A00&end=2025-01-11T00%3A00%3A00%2B02%3A00');
+
+let Course=[];
+async function getCourse(link) {
+  const response=await fetch(link)
+  const data=await response.json();
+  data.events.forEach((event,index) =>{
+
+  const start=event.start
+  const end=event.end
+
+  // Load the title HTML into Cheerio
+  const $ = cheerio.load(event.title);
+  
+  // Select the <a> element and get the data-title attribute
+  let dataTitle = $($('a').attr('data-title')).find('a').text();
+  if(!dataTitle)dataTitle=event.title;
```

Comment:
- always write if's in the following format

if(condition) {
   function body
}